libsel4vm/src/arch/arm/vgic/vgic.c

```diff
@@ -121,13 +121,11 @@
 #define IRQ_IDX(irq) ((irq) / 32)
 #define IRQ_BIT(irq) (1U << ((irq) % 32))
 
-struct virq_handle {
+typedef struct virq_handle {
     int virq;
     irq_ack_fn_t ack;
     void *token;
-};
-
-typedef struct virq_handle *virq_handle_t;
```

Comment:
This typedef isn't allowed by the style guide: https://docs.sel4.systems/processes/style-guide.html#naming-of-symbols-variables-types-and-type-aliases-typedef.

Reply:
goot point, weill remove this from the code then in a separate PR.